Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:latest
+
+WORKDIR /Desktop/www/project_1
+
+RUN apt-get update
+
+RUN DEBIAN_FRONTEND=noninteractive DEBCONF_NOWARNINGS=yes apt-get -y install vim
+
+RUN DEBIAN_FRONTEND=noninteractive DEBCONF_NOWARNINGS=yes apt-get -y install build-essential
+
+COPY scripts .
+
+RUN ./ird.sh ./
+
+RUN ./reversenumber.sh 517
```

Comment:
I think this will not work. You have copied a directory. By intent. your scripts are located in `/Desktop/www/project_1/scripts` directory. But you working directory is /Desktop/www/project_1